app/assets/stylesheets/reviews.scss

```diff
@@ -1,14 +1,142 @@
-// Place all the styles related to the reviews controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
+.main-content {
```

Comment:
This was moved here from `app/assets/stylesheets/users/reviews.scss` with some minor changes.